README.md

```diff
@@ -71,9 +71,9 @@ make debug
 
 This will run the app with `DEBUG` mode on, showing full error messages in-browser when they occur.
 
-When running in development mode it is expected that you have a database running locally, accessable via `localhost:27017`.
+When running in development mode it is expected that you have a database running locally, accessible via `localhost:27017`.
 
-To produce some data for the flask app to display, follow the instructions in the following section.
+To produce some data for the Flask app to display, follow the instructions in the following section.
```

Comment:
@buckley-w-david can you please update this line to link to the proper place? "the following section" should be something else but I'm not positive where that doc is.